### DISCUSSION TITLE
print value at a pointer as string "gdb x/s pointer"

### DISCUSSION THREAD

**Discussion by k-x7** [go-delve/delve#3614] [Q&A] [answered]
Hi all

How I print the value at a memory location as string?
The binary has been build with strip flags `-ldflags="-s -w"`, args will not works.

A pointer `0x000000c000012090` point to a byte slice 
```[]byte(`{"A": 1, "B": 2}`)```

examine the memory will give me what I need but in hex:
```
(dlv) x -size 8 -count 3 0x000000c000012090
0xc000012090:   0x2c31203a2241227b   0x7d32203a22422220   0x0000000000000000
```

how to print the value in a string format? or as []byte with len?

**Comment by aarzilli** [comment, marked as answer]
Nobody has implemented this.

**Comment by k-x7** [comment]
thank you for the answer, another question can I run a go or python code or any script using the `on` command?

**Reply by aarzilli** [reply]
No, the `on` command is pretty limited at the moment but it would be a good idea to extend starlark scripts to it.
